openapi/openapi_client.go

```diff
@@ -83,6 +86,9 @@ func (o *ProviderClient) performRequest(method httpMethodSupported, resourceURL
 	o.appendOperationHeaders(operation.HeaderParameters, o.providerConfiguration, reqContext.headers)
 	log.Printf("[DEBUG] Performing %s %s", method, reqContext.url)
```

Comment:
Removing the two above lines do not break any tests either.

Reply:
+1 let's add coverage for that while we are on this